### PR TITLE
Removed Nette\Object

### PR DESCRIPTION
**Pull request by ondrakub**
Removed Nette\Object for support of PHP 7.2

### CODE REVIEW

**Comment by ondrakub** [comment]
@f3l1x Do you approve this if it is all right? :) I dont want to hurry, but we're waiting for this in our company.

File: src/Gopay/Gopay.php [hunk 1]
```diff
@@ -13,9 +13,11 @@
  * @property-read GopaySoap $soap
  * @property-read GopayHelper $helper
  */
-class Gopay extends Object
+class Gopay
 {
 
+	use Nette\SmartObject;
```
Comment (f3l1x) [inline src/Gopay/Gopay.php]: Please import this class.

**Comment by f3l1x** [comment]
It's great, one last thing. Add PHP 7.2 to Travis check.

**Comment by f3l1x** [comment]
Bumped in v3.2.0

**Comment by natocTo** [comment]
How long it usually take to reflect a new version on packagist? No new version there, it is weird.

**Comment by f3l1x** [comment]
It's there. I've had to change `New BSD` to `BSD-3-Clause` in composer.json.

**Pull request merged**